<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OSC_GeneratorDockWidgetBase</class>
 <widget class="QDockWidget" name="OSC_GeneratorDockWidgetBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>386</width>
    <height>290</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add Vehicles</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QGridLayout" name="gridLayout">
    <item row="8" column="0">
     <widget class="QPushButton" name="vehicle_labels">
      <property name="text">
       <string>Labels</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1" colspan="2">
     <widget class="QLineEdit" name="vehicle_init_speed">
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item row="8" column="1" colspan="2">
     <widget class="QPushButton" name="add_vehicle_button">
      <property name="text">
       <string>Insert</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QCheckBox" name="vehicle_orientation_use_lane">
      <property name="text">
       <string>Use lane heading</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="1" colspan="2">
     <widget class="QComboBox" name="vehicle_selection">
      <item>
       <property name="text">
        <string>Audi A2</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Audi eTron</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Audi TT</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>BH Crossbike</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>BMW Grandtourer</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>BMW iSetta (CARLA 0.9.12)</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>BMW iSetta (CARLA 0.9.11)</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Carla Cola Truck</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Chevrolet Impala</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Citroen C3</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Diamondback Century</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Dodge Charger Police (CARLA 0.9.12)</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Dodge Charger Police (CARLA 0.9.11)</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Gazelle Omafiets</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Harley Davidson Low Rider</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Jeep Wrangler</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Kawasaki Ninja</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Lincoln MKZ 2017 (CARLA 0.9.12)</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Lincoln MKZ 2017 (CARLA 0.9.11)</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Lincoln MKZ 2020 (CARLA 0.9.12)</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Lincoln MKZ 2020 (CARLA 0.9.11)</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Mercedes Benz Coupe (CARLA 0.9.12)</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Mercedes Benz Coupe (CARLA 0.9.11)</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Mini Cooper ST (CARLA 0.9.12)</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Mini Cooper ST (CARLA 0.9.11)</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Ford Mustang (CARLA 0.9.12)</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Ford Mustang (CARLA 0.9.11)</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Nissan Micra</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Nissan Patrol</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Seat Leon</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Tesla Cybertruck</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Tesla Model 3</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Toyota Prius</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Volkswagen T2</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Yamaha YZF</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Firetruck (CARLA 0.9.12)</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Mini Cooper 2021 (CARLA 0.9.12)</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Nissan Patrol 2021 (CARLA 0.9.12)</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Ambulance (CARLA 0.9.12)</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Mercedes Sprinter (CARLA 0.9.12)</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="vehicle_orientation">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>90</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="vehicle_orientation_label">
      <property name="text">
       <string>Orientation</string>
      </property>
     </widget>
    </item>
    <item row="7" column="0" colspan="3">
     <widget class="QGroupBox" name="agent_group">
      <property name="title">
       <string>Agent Selection</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="1" column="0">
        <widget class="QComboBox" name="agent_selection">
         <item>
          <property name="text">
           <string>simple_vehicle_control</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>external_control</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>User Defined</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QCheckBox" name="agent_attach_camera">
         <property name="text">
          <string>Attach Camera</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLineEdit" name="agent_user_defined">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="placeholderText">
          <string>user-defined agent</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QCheckBox" name="vehicle_is_hero">
      <property name="text">
       <string>Ego Vehicle</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="vehicle_selection_label">
      <property name="text">
       <string>Vehicle</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="vehicle_init_speed_label">
      <property name="text">
       <string>Init Speed</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>vehicle_selection</tabstop>
  <tabstop>add_vehicle_button</tabstop>
  <tabstop>vehicle_orientation</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
